### DISCUSSION TITLE
HSL support

### DISCUSSION THREAD

**Discussion by jansedlon** [mantinedev/mantine#2607] [Feature requests]
Hello, I found out that if my theme uses HSL colors, sometimes it doesn't work. For example when using `<ThemeIcon variant="white">...</ThemeIcon>`, background of the icon is just black.. That's because `rgba` function doesn't support conversion from HSL to RGBA. Would you be interested in this feature? I can implement it

**Comment by rtivital** [comment]
Yeah, I'm fine with that, but keep in mind that there should not be any third-party color libraries
